NosTests/Service/DatabaseCleanerTests.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
+
+final class DatabaseCleanerTests: CoreDataTestCase {
+    
+    func test_emptyDatabase() async throws {
+        try await DatabaseCleaner.cleanupEntities(before: Date.now, for: KeyFixture.alice.publicKeyHex, in: testContext)
```

Comment:
Do we need an assert in here? Or is this just to verify that cleanupEntities doesn't crash?

Reply:
Yeah this is just to make sure it doesn't crash. I suppose we could assert that cleanupEntities didn't _create_ any entities but that seems a little crazy. But why not, I can do that quickly.